dockerfiles/init/modules/rsyslog/templates/postgre.conf.erb

```diff
@@ -0,0 +1,3 @@
+$FileCreateMode 0644
```

Comment:
name of the file should be postgresql.conf.erb maybe ?

and /var/logs/postgresql/postgresql.log ?